frontend/src/metabase/admin/settings/app/components/SettingsEditor/SettingsEditor.jsx

```diff
@@ -81,18 +81,28 @@ class SettingsEditor extends Component {
   }
 
   /**
-   * @param {Object} setting
+   * @param {Object|import("metabase-types/api").SettingKey} settingOrKey
```

Comment:
this makes this SO MUCH harder to reason about by introducing an inconsistent pattern 😢

Reply:
I'll revert this as requested.